ga/23.0.0.6/full/Dockerfile.ubuntu.ibmjava8

```diff
@@ -12,38 +12,29 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM websphere-liberty:23.0.0.6-kernel-java8-ibmjava AS installBundle
+FROM websphere-liberty:kernel-java8-ibmjava
+
+USER root
```

Comment:
How did this work before without USER root?

Reply:
I think that's a question for @leochr, and whether we need it for all the full images, including java8

Reply:
`USER root` shouldn't be used, especially with single-stage Dockerfile as that'll switch the default user of Liberty full image. I believe it was added as part of multi-stage Dockerfiles changes, but I don't think it's needed to perform the operations in the full image.